bot/cogs/embeds.py

```diff
@@ -0,0 +1,323 @@
+import textwrap
+from collections import defaultdict
+
+from discord import Colour, Embed, TextChannel
+from discord.ext.commands import Cog, Context, command, group, ColourConverter
+
+from bot import constants
+from bot.bot import Bot
+from bot.cogs.moderation.modlog import ModLog
+from bot.decorators import with_role
+
+prefix = constants.Bot.prefix
+
+
+class Embeds(Cog):
+    """
+    A cog which allows user to send messages using the bot
+    """
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+        self.embed_mode = defaultdict(bool)
+        self.embed = {}
+        self.embed_field_id = defaultdict(lambda: -1)
+
+    @property
+    def mod_log(self) -> ModLog:
+        """Get currently loaded ModLog cog instance."""
+        return self.bot.get_cog("ModLog")
+
+    # region: embed mode
+
+    @command(aliases=["embedcreate"])
+    @with_role(*constants.MODERATION_ROLES)
```

Comment:
It would be easier to read if you directly imported `MODERATION_ROLES` from `bot.constants` instead of using `constants.MODERATION_ROLES`
```suggestion
    @with_role(*MODERATION_ROLES)
```

Reply:
It would be handy to not use `constants.XXX` at all and simply import the needed constants directly

Reply:
Resolved in 930639c6a1551f5ee9f8a07264268270ec29d471